app/assets/javascripts/address-verification.js.coffee.erb

```diff
@@ -58,14 +42,38 @@ $(document).on 'ready page:load', ->
     key: smarty_key, debug: false, enforceVerification: true, autoVerify: false,
     fieldSelector: 'input[type=text], input:not([type]), textarea, select:not([name*="country_id"])'
 
-  $(country_selector).trigger 'change'
+  for country in $ 'select[name*="country_id"]'
+    validator = smarty_validator_for country
+    us_address = $(country).find('option:selected').text().match /united states/i
```

Comment:
No, `val` will return the `id` of the option. We want to check if the label is US.